app/src/main/java/org/lineageos/twelve/fragments/AlbumFragment.kt

```diff
@@ -140,6 +146,35 @@ class AlbumFragment : Fragment(R.layout.fragment_album) {
                                 thumbnailImageView.setImageBitmap(album.thumbnail)
                             }
 
+                            artistNameTextView.text = album.artistName
+                            artistNameTextView.setOnClickListener {
+                                findNavController().navigate(
+                                    R.id.action_albumFragment_to_fragment_artist,
+                                    ArtistFragment.createBundle(album.artistUri)
+                                )
+                            }
+
+                            album.year?.also { year ->
+                                yearTextView.isVisible = true
+                                yearTextView.text = year.toString()
+                            } ?: run {
+                                yearTextView.isVisible = false
+                            }
+
+                            launch(Dispatchers.IO) {
```

Comment:
Not worth it
The "context switch" is likely more expensive then the actual operations

Reply:
Done